modules/ibc-hooks/simapp/app.go

```diff
@@ -538,49 +506,32 @@ func NewSimApp(
 	// 'ibc' module - depends on
 	// 1. 'staking'
 	// 2. 'upgrade'
-	// 3. 'capability'
 	// 4. 'gov'
 	// 5. 'params'
 	app.keys[ibcexported.StoreKey] = storetypes.NewKVStoreKey(ibcexported.StoreKey)
 	app.IBCKeeper = ibckeeper.NewKeeper(
 		appCodec,
-		keys[ibcexported.StoreKey],
+		runtime.NewKVStoreService(keys[ibcexported.StoreKey]),
 		app.GetSubspace(ibcexported.ModuleName),
-		app.StakingKeeper,
 		app.UpgradeKeeper,
-		scopedIBCKeeper,
 		govModAddress,
 	)
 
-	govLegacyRouter.AddRoute(ibcexported.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper))
 	simModules = append(simModules, ibc.NewAppModule(app.IBCKeeper))
 
 	// 'ibcfeekeeper' module - depends on
 	// 1. 'bank'
 	// 2. 'auth'
 	// 3. 'ibc channel'
 	// 4. 'ibc port'
-	app.IBCKeeper = ibckeeper.NewKeeper(
```

Comment:
The comment here was wrong, this is not the fee keeper

Reply:
weird, ibckeeper already existed: 
https://github.com/cosmos/ibc-apps/pull/272/files?diff=unified#diff-c7aeadef5480ea593dd152867ab52c7f60e2df19c2ada3766951cd5f3d54bc1aR512
already exists before